DEVGUIDE.md

```diff
@@ -0,0 +1,50 @@
+
+# Contents
+1. [Build & Run tests locally](#build--test)
+2. [Build API Reference Documentation](#build-api-reference-documentation)
+3. [Contributing](CONTRIBUTING.md)
+4. [Creating a release](#releasing)
+
+## Build & Test
+
+Prerequisites:
+- [Docker](https://docker.com/) / [JSON API Server](https://github.com/stargate/jsonapi)
+
+### Build
+- Run `npm install && npm run build`
```

Comment:
Maybe this is overdoing it, but if we make all these commands in code blocks then there will be a copy button, right?